datastore/integration-test/test_api.py

```diff
@@ -150,14 +150,14 @@ def test_from_a_single_collection_get_a_single_location_which_does_not_exist():
 
     expected_json = load_json("response/404_not_found.json")
 
-    assert actual_response.status_code == 404
+    assert actual_response.status_code == 400
```

Comment:
Why did this change?
The file is also still named with 404.

Reply:
It changed due to verify_parameter_name, I updated the filename.